### PR TITLE
Add InstallApps

### PR DESCRIPTION
**Pull request by rossf7**
Towards https://github.com/giantswarm/giantswarm/issues/13206

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by rossf7** [comment]
This is a new library for creating app CRs in integration tests.

It will be used in projects with Go integration tests and in apptestctl to install chartmuseum.

File: apptest.go [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+	for _, app := range apps {
+		a.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating %#q app cr", app.Name))
+
+		appCR := &v1alpha1.App{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      app.Name,
+				Namespace: app.Namespace,
+				Labels: map[string]string{
+					// Processed by app-operator-unique.
+					label.AppOperatorVersion: "0.0.0",
```
Comment (tomahawk28) [inline apptest.go:123]: can we keep "0.0.0" as constant as well?

File: apptest.go [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+	for _, app := range apps {
+		a.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating %#q appcatalog cr", app.CatalogName))
+
+		appCatalogCR := &v1alpha1.AppCatalog{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: app.CatalogName,
+				Labels: map[string]string{
+					// Processed by app-operator-unique.
+					label.AppOperatorVersion: "0.0.0",
```
Comment (tomahawk28) [inline apptest.go:86]: Here, too.

**Review by tomahawk28 — APPROVED** [review]
LGTM, just two comments.

File: apptest.go (hunk 1 above)
Comment (rossf7) [reply apptest.go]: Yes good point. Done.

**Pull request merged**